### PR TITLE
test(test): test

### PR DESCRIPTION
**Pull request by gaonita**
test aws

**Describe pull-request**  
_Describe what the pull-request is about_

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: #

**How to test**  
_Add description how to test if possible_
1. Go to...
2. Check in...
3. Run ...

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
@gaonita  - Did you just make Storybook spins on AWS? If so, awesome, really awesome work!

**Comment by gaonita** [comment]
> it doesn't work perfectly yet🙈  therefore this test PR is messy. but I'm trying! anyway thanks! :D

**Comment by mmexvr** [comment]
Super cool! Would be awesome if this worked. 🚀  ☁️ 

I have two things to look at here :)
- On netlify we don't use start command. We have a separate command called netlify that only builds and uses the dist from storybook, under component folder, for the preview.
- Keep track how the cost work for that service, cost management service in AWS you can se everything

File: components/.storybook/preview.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { setCustomElements, addDecorator } from '@storybook/web-components';
-import { withThemes } from 'storybook-addon-themes/html';
+// import { withThemes } from 'storybook-addon-themes/html';
 import customElements from '../dist/collection/custom-elements.json';
 
-addDecorator(withThemes);
+// addDecorator(withThemes);
```
Comment (theJohnnyMe) [inline components/.storybook/preview.js:5]: I noticed that error is not related to the content, but to the storybook and aws. It felt that some of the code is not aligned with standard, while Netlify does not complain, aws seems to do. Anyway, keep on, it looks like you are on a good way to solve it. I would like to hear your impressions of it tomorrow during check in.

**Pull request closed without merging**